### ISSUE TITLE
css not imported in minified index.js

### ISSUE THREAD

**Issue by daskyrk** [otakustay/react-diff-view#17]
it seems the css file is not imported in index.js:
```javascript
import { parseDiff, Diff, Hunk } from 'react-diff-view';
import 'react-diff-view/index.css'; // have to import standalone
```
by the way, it's better to remove `console.log` in the minified index.js

**Comment by otakustay** [comment]
CSS is not imported by default, for a bundled third party lib, I don't want to import a css in a "javascript way", it causes the bundle to include a script to create `<style>` element which is ugly

For the `console.log` issue, I'll have a look at it and publish a patch version to remove all

**Comment by daskyrk** [comment]
Thanks

发自我的 iPhone

**Comment by otakustay** [comment]
Published `1.8.0` to eliminate console statements, also `parseDiff` now accepts unidiff as input
